src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -135,21 +135,17 @@ public TextWriter Log
 
         private CORINFO_MODULE_STRUCT_* _methodScope; // Needed to resolve CORINFO_EH_CLAUSE tokens
 
-        private bool _isFallbackBodyCompilation; // True if we're compiling a fallback method body after compiling the real body failed
-
-        private void CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL methodIL = null)
+        private void CompileMethodInternal(IMethodNode methodCodeNodeNeedingCode, MethodIL methodIL)
```

Comment:
Nit: we can also get rid of the first parameter because that one is available as `_methodCodeNode` (parts of this method already use that...)